code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -218,54 +218,73 @@
 	glass_icon_state  = "glass_clear"
 	glass_name = "glass of holy water"
 	glass_desc = "A glass of holy water."
+	metabolization_rate = 0.4 //a fixed consumption rate to prevent balancing going out of whack
 	self_consuming = TRUE //divine intervention won't be limited by the lack of a liver
+	var/seizure = FALSE //has Nar'Sie chewed you out during this deconversion process yet?
```

Comment:
This isn't dmdoc compliant, please follow our [code contribution guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine) regarding class-level variable declarations and their documentation.